Doc/library/decimal.rst

```diff
@@ -571,29 +585,11 @@ Decimal objects
       >>> Decimal(321).exp()
       Decimal('2.561702493119680037517373933E+139')
 
-   .. method:: from_float(f)
+   .. classmethod:: from_float(f)
 
-      Classmethod that converts a float to a decimal number, exactly.
-
-      Note `Decimal.from_float(0.1)` is not the same as `Decimal('0.1')`.
-      Since 0.1 is not exactly representable in binary floating point, the
-      value is stored as the nearest representable value which is
-      `0x1.999999999999ap-4`.  That equivalent value in decimal is
-      `0.1000000000000000055511151231257827021181583404541015625`.
-
-      .. note:: From Python 3.2 onwards, a :class:`Decimal` instance
-         can also be constructed directly from a :class:`float`.
-
-      .. doctest::
-
-          >>> Decimal.from_float(0.1)
-          Decimal('0.1000000000000000055511151231257827021181583404541015625')
-          >>> Decimal.from_float(float('nan'))
-          Decimal('NaN')
-          >>> Decimal.from_float(float('inf'))
-          Decimal('Infinity')
-          >>> Decimal.from_float(float('-inf'))
-          Decimal('-Infinity')
+      Alternative constructor that only accepts instances of :class:`float` or
+      :class:`int`. A legacy method which largely exists for backwards
```

Comment:
> How does [8f537ea](https://github.com/python/cpython/commit/8f537ea9e72a3c10b116ae2b4ec4989f8f53c089) look?

Looks good to me! At nitpick level, please can we do this:
```suggestion
      :class:`int`. This is a legacy method which largely exists for backwards
```

Reply:
Thanks, done: https://github.com/python/cpython/pull/92421/commits/44357721f48839b87e88f4846d56daf0710d673b